src/qml/editorwidgets/TextEdit.qml

```diff
@@ -8,7 +8,7 @@ import "."
 EditorWidgetBase {
   id: topItem
 
-  property bool isEditable: isEnabled && value && typeof(value) !== 'object'
+  property bool isEditable: isEnabled && typeof(value) !== 'object'
```

Comment:
Beware that `typeof(null) === 'object'`

Reply:
Ok, I'll need to go at it another way, something was germinating at this very moment

Reply:
you can always do `value === null || typeof(value) !== 'object'`

Reply:
Yeah but ultimately I think it makes more sense to rely on field type here.